### PR TITLE
PMM-10051-add-aurora-remote-tests

### PR DESCRIPTION
**Pull request by beatahandzelova**
pmm2-ui-tests: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/19054/tests
pmm2-upgrade-tests: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-upgrade-tests/detail/pmm2-upgrade-tests/17068/tests

### CODE REVIEW

File: tests/verifyAuroraRemoteInstance_test.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Data(instances)
+  .Scenario('PMM-T1295 Verify Aurora instance metrics @instances', async ({ I, current, grafanaAPI }) => {
+    const { instance_id } = current;
+
+    // Waiting for metrics to start hitting for remotely added services
+    I.wait(10);
+
+    // verify metric for client container node instance
+    const response = await grafanaAPI.checkMetricExist(metric, {
```
Comment (puneet0191) [inline tests/verifyAuroraRemoteInstance_test.js:60]: any reason why you are not checking for RDS related metrics, Aurora related metrics?

File: tests/verifyAuroraRemoteInstance_test.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      I.amOnPage(qanPage.url);
+      qanOverview.waitForOverviewLoaded();
+      await adminPage.applyTimeRange('Last 12 hours');
+      qanOverview.waitForOverviewLoaded();
+      qanFilters.waitForFiltersToLoad();
+      await qanFilters.applySpecificFilter(instance_id);
+      qanOverview.waitForOverviewLoaded();
+      const count = await qanOverview.getCountOfItems();
+
+      assert.ok(count > 0, `The queries for service ${instance_id} instance do NOT exist, check QAN Data`);
```
Comment (puneet0191) [inline tests/verifyAuroraRemoteInstance_test.js:117]: if I read this error message on Allure report for a failing test, I have no idea what or which Aurora is failing, what is instance_id? the one generated by PMM Managed?

File: tests/pages/api/addInstanceAPI.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ module.exports = {
     };
 
     const headers = { Authorization: `Basic ${await I.getAuth()}` };
-    const resp = await I.sendPostRequest('/v1/management/MySQL/Add', body, headers);
+    const resp = await I.sendPostRequest('v1/management/MySQL/Add', body, headers);
```
Comment (puneet0191) [inline tests/pages/api/addInstanceAPI.js:51]: how was this working before?

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/verifyAuroraRemoteInstance_test.js (hunk 1 above)
Comment (beatahandzelova) [reply tests/verifyAuroraRemoteInstance_test.js]: We don't have aurora metrics for Aurora3 due to a bug. So I left the test which checks 'mysql_global_status_max_used_connections' for both Aurora2 and 3 and added another test which checks 'mysql_global_status_aurora_total_op_memory' only for Aurora2 until the bug is fixed.

**Review by peterSirotnak — APPROVED** [review]
LGTM
https://pmm.cd.percona.com/job/pmm2-ui-tests/19951/allure/#suites/38bbc8f727415fe687fd4a7c95b12aab/ed832b1d09584bef/

**Pull request merged**